Makefile

```diff
@@ -1,15 +1,17 @@
 export CARGO_PROFILE_RELEASE_CODEGEN_UNITS = 1
 export CARGO_PROFILE_RELEASE_LTO = fat
 export DOCKER_BUILDKIT = 1
-export RUSTFLAGS = -D warnings
 export NEAR_RELEASE_BUILD = no
 export CARGO_TARGET_DIR = target
 
+export RUSTFLAGS = -D warnings
+ifeq ($(shell uname -m),x86_64)
+  export RUSTFLAGS := ${RUSTFLAGS} -Ctarget-feature=+sse4.1,+sse4.2
+endif
```

Comment:
So I’m always unsure about such changes to Makefile.  Specifically, this is GNU Make syntax and it’s my understanding that we support builds on Macs and thus must work with intersection supported by BSD Make.

Reply:
In fact MacOS uses GNU make so no problem with that. .cargo/config.toml doesn't apply for some reason. I'm trying to figure out why that's the case.

Reply:
Okay I found the issue: 'RUSTFLAGS = -D warnings' was overriding rustflags coming from config so I moved it to the config as well.